src/Components/CommonComponents/LoaderSpinner.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import Loader from 'react-loader-spinner';
+import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css';
+
+// if recives props.full shows a fullscreen loader component
+const LoaderSpinner = ({ full = false }) => {
+  if (full) {
+    return (
+      <Box
+        sx={{
+          height: '100vh',
+          width: '100%',
+          position: 'absolute',
+          backgroundColor: 'rgba(255,255,255,0.99)',
+          zIndex: '3',
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}>
```

Comment:
please move this sx style into a const to reuse on both styles the styles that you can reuse.

```
const spinnerStyle = {
         height: '100vh',
         width: '100%',
         display: 'flex',
          flexDirection: 'column',
          justifyContent: 'center',
          alignItems: 'center',
};
const fullSize = {
  position: 'absolute',
          backgroundColor: 'rgba(255,255,255,0.99)',
          zIndex: '3',
}
const relativeSize = {
          position: 'relative',
}

const LoaderSpinner = ({ isFullscreenLoader= false }) => isFullScreenLoader ? (<Box sx={{...spinnerStyle, ...fullSize}}>...</>) : ...
```